packages/app/src/display/style/components/ui/_overlay.scss

```diff
@@ -111,15 +111,63 @@
 }
 
 .ready-overlay-content {
-    .h2h {
-        margin-bottom: 0.5em;
-        font-size: 1.4em;
+    .h2h-header {
+        margin-bottom: 0.25em;
+        font-size: 1.5em;
         text-align: center;
+    }
 
-        .opponent {
-            font-weight: bold;
-            color: #ff6464;
+    .inner-profile-box {
+        display: flex;
+        flex-direction: row;
+    }
+
+    .name-and-health {
+        display: flex;
+        flex-direction: column;
+        font-family: Arial, sans-serif;
+        text-transform: capitalize;
+
+        &.right {
+            justify-content: right;
+            align-items: right;
+            text-align: right;
         }
+
+        .healthbar-container {
+            width: 12em;
```

Comment:
When sizing components in CSS, it's often better to use `flex` and size things proportionally rather than a specific width.

By giving the healthbar container a fixed width in terms of units rather than %s, there's a natural limitation on the number screen sizes where this will work as you expect. A better solution could be to just make the `.healthbar-container` have a width of `100%` and let it scale up to the full width of the `name-and-health-container` - but that's only a guess from looking at the code

I'm happy to merge this for now because `width` isn't a bad solution, but it might be good to come back to this and try to future proof it